package.json

```diff
@@ -27,8 +27,8 @@
         "displayName": "UI: Tags module is enabled"
       },
       {
-        "permissionName": "settings.tags.enabled",
```

Comment:
you can't delete such permissions, do you know any module where it's deleted? you can check stripes docs, they are required

Reply:
thanks. I was looking at `module.tags.enabled`

Reply:
from my perspective this PR is still wrong. It looks like yo need new permission, created according to the guidelines with translation,  based on "settings.tags.enabled" permission. 
at the same time "settings.tags.enabled" should become visible: false

Reply:
fixed